src/WebdriverClassicDriver.php

```diff
@@ -1,21 +1,1345 @@
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\Remote\RemoteWebElement;
+use Facebook\WebDriver\WebDriverBy;
+use Facebook\WebDriver\WebDriverDimension;
+use JetBrains\PhpStorm\Language;
+use JsonException;
+use Throwable;
 
 class WebdriverClassicDriver extends CoreDriver
 {
+    public const DEFAULT_BROWSER = 'chrome';
+
+    public const DEFAULT_CAPABILITIES = [
+        'default' => [
+            'platform' => 'ANY',
+            'name' => 'Behat Test',
+            'deviceOrientation' => 'landscape',
+            'deviceType' => 'desktop',
+        ],
+        'chrome' => [
+            'goog:chromeOptions' => [
+                'excludeSwitches' => ['enable-automation'],
+            ],
+        ],
+        'firefox' => [
+        ],
+    ];
+
+    private const W3C_WINDOW_HANDLE_PREFIX = 'w3cwh:';
+
+    private ?RemoteWebDriver $webDriver = null;
+
+    private string $browserName;
+
+    private DesiredCapabilities $desiredCapabilities;
+
+    private bool $started = false;
+
+    private array $timeouts = [];
+
+    private Escaper $xpathEscaper;
+
+    private string $webDriverHost;
+
+    private ?string $initialWindowName = null;
+
+    /**
+     * @throws DriverException
+     */
+    public function __construct(
+        string  $browserName = null,
+        array   $desiredCapabilities = null,
+        string  $webDriverHost = null,
+        Escaper $xpathEscaper = null
+    ) {
+        $this->browserName = $browserName ?? self::DEFAULT_BROWSER;
+        $this->setDesiredCapabilities($this->initCapabilities($desiredCapabilities ?? []));
+        $this->webDriverHost = $webDriverHost ?? 'http://localhost:4444/wd/hub';
+        $this->xpathEscaper = $xpathEscaper ?? new Escaper();
+    }
+
+    // <editor-fold desc="Implementation">
+
+    /**
+     * {@inheritdoc}
+     * @throws DriverException
+     */
+    public function start(): void
+    {
+        try {
+            $this->webDriver = RemoteWebDriver::create($this->webDriverHost, $this->desiredCapabilities);
+            $this->applyTimeouts();
+            $this->initialWindowName = $this->getWindowName();
+            $this->started = true;
+        } catch (Throwable $e) {
+            throw new DriverException("Could not open connection: {$e->getMessage()}", 0, $e);
+        }
+    }
+
+    /**
+     * Sets the timeouts to apply to the webdriver session
+     *
+     * @param array $timeouts The session timeout settings: Array of {script, implicit, page} => time in milliseconds
+     * @throws DriverException
+     * @api
+     */
+    public function setTimeouts(array $timeouts): void
+    {
+        $this->timeouts = $timeouts;
+
+        if ($this->isStarted()) {
+            $this->applyTimeouts();
+        }
+    }
+
+    /**
+     * {@inheritdoc}
+     */
     public function isStarted(): bool
     {
-        return false;
+        return $this->started;
```

Comment:
I also suggest creating a private helper

```php
private function getWebDriver(): WebDriver
{
    if ($this->webDriver === null) {
        throw new DriverException('The driver is not started.');
    }

    return $this->webDriver;
}
```

Using this private helper in methods would allow to provide a better feedback than a fatal error when calling a method on a non-started driver, and it will help once we enable static analysis (for now, I added phpstan only on the Mink repo, not yet on drivers, but that's a next step).